filebeat/docs/modules.asciidoc

```diff
@@ -9,4 +9,6 @@ modules.
 
 Filebeat modules require Elasticsearch 5.2 or later.
 
+NOTE: Bug fixes, enhancements, and new Filebeat modules are no longer implemented. We recommend that you {fleet-guide}/migrate-beats-to-agent.html[migrate to {agent}], whose integrations are actively maintained.
```

Comment:
@natasha-moore-elastic thanks for raising the PR. I've added a suggestion below to flesh out the text. FYI @SubhrataK as this text will cover o11y modules too. 

While Filebeat modules are still supported, Elastic Agent integrations are recommended over Filebeat modules. Integrations are streamlined way to connect your data to the Elastic Stack and are available for popular services and platforms, across security and observability, the full list is available [here](https://www.elastic.co/integrations/data-integrations). 

Please refer to the [Beats vs Elastic Agent comparison](https://www.elastic.co/guide/en/fleet/current/beats-agent-comparison.html) documentation for more information.